internal/command/postgres/failover.go

```diff
@@ -339,10 +360,19 @@ func handleFlexFailoverFail(ctx context.Context, machines []*fly.Machine) (err e
 	return nil
 }
 
-func pickNewLeader(ctx context.Context, app *fly.AppCompact, machinesWithinPrimaryRegion []*fly.Machine) (*fly.Machine, error) {
+func pickNewLeader(ctx context.Context, app *fly.AppCompact, primaryCandidates []*fly.Machine, secondaryCandidates []*fly.Machine, allowSecondaryRegion bool) (*fly.Machine, error) {
 	machineReasons := make(map[string]string)
 
-	for _, machine := range machinesWithinPrimaryRegion {
+	// We should go for the primary canddiates first, but the secondary candidates are also valid
+	var candidates []*fly.Machine
+	switch allowSecondaryRegion {
+	case true:
+		candidates = append(primaryCandidates, secondaryCandidates...)
+	case false:
+		candidates = primaryCandidates
+	}
```

Comment:
`switch` on a boolean seems unusual. Why not just use an `if`/`else`?

Reply:
genuinely don't know why i wrote it like this. if else is a few less lines so i'll switch it to that